pkg/apis/policy/v1alpha1/clusterimagepolicy_validation.go

```diff
@@ -154,19 +160,15 @@ func (key *KeyRef) Validate(ctx context.Context) *apis.FieldError {
 
 func (keyless *KeylessRef) Validate(ctx context.Context) *apis.FieldError {
 	var errs *apis.FieldError
-	if keyless.URL == nil && keyless.Identities == nil && keyless.CACert == nil {
-		errs = errs.Also(apis.ErrMissingOneOf("url", "identities", "ca-cert"))
+	if keyless.URL == nil && keyless.CACert == nil {
+		errs = errs.Also(apis.ErrMissingOneOf("url", "ca-cert"))
 	}
 
 	// TODO: Are these really mutually exclusive?
 	if keyless.URL != nil && keyless.CACert != nil {
 		errs = errs.Also(apis.ErrMultipleOneOf("url", "ca-cert"))
 	}
 
-	if keyless.Identities != nil && len(keyless.Identities) == 0 {
```

Comment:
Question: won't we need to handle this condition ?

Reply:
It's marked as optional and it's fine to say that you don't care who signed (or attested something) :) If you don't give these, then as long as somebody signed or attested will be accepted :)

Reply:
Gotcha! 👍🏻